src/common/transformations/include/transformations/common_optimizations/move_eltwise_up_data_movement.hpp

```diff
@@ -11,10 +11,25 @@
 namespace ov {
 namespace pass {
 
-class TRANSFORMATIONS_API MoveEltwiseUpThroughDataMov : public ov::pass::MatcherPass {
+class TRANSFORMATIONS_API MoveEltwiseUpThroughDataMovScalar : public ov::pass::MatcherPass {
+public:
+    OPENVINO_RTTI("MoveEltwiseUpThroughDataMovScalar", "0");
+    MoveEltwiseUpThroughDataMovScalar(std::vector<DiscreteTypeInfo> allowed_data_movement_ops);
+};
+
+class TRANSFORMATIONS_API MoveEltwiseUpThroughDataMovPerChannel : public ov::pass::MatcherPass {
```

Comment:
could you add a description?
it would be great if you could add a small example of what happens in the transformation

Reply:
Added